src/inverseRobot.js

```diff
@@ -24,7 +24,17 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const reverseRobot = {};
+
+  for (const key in robot) {
+    if (!reverseRobot[robot[key]]) {
```

Comment:
![exp](https://user-images.githubusercontent.com/108527147/189926758-7de4dc83-c58b-42fa-9538-d09ae7e10175.JPG)
According to the checklist, it's not the best solution, sadly...